### DISCUSSION TITLE
Graphics Issues - Surface Pro 2

### DISCUSSION THREAD

**Discussion by ZProLegend007** [sebanc/brunch#1369] [Q&A]
Hi There!
I'm relatively new to changing operating systems on different computers but i'm getting the hang of things. I just installed Chrome os on my Surface pro 2 and i'm having a few graphical issues, I've been using asphalt 9 to test my fixes to little or no avail. I'm not sure if it is officially supported on this laptop but if there is any way to fix this it would be greatly appreciated. I will attach some images below to compare what this loading symbol does and should look like.

![image](https://user-images.githubusercontent.com/97338659/148640982-3f60d410-a7e5-4268-ba0e-9a17c55e4f78.png)
![image](https://user-images.githubusercontent.com/97338659/148640985-8f77e2a8-bc61-45f0-9de8-2b167b60c9a6.png)

**Comment by tejasraman** [comment]
Are you using the linux-surface boot option? This enables Surface-specific kernel patches so that Surface devices work correctly

**Reply by ZProLegend007** [reply]
I'm not sure, could you tell me how to use that? Everything else works pretty well other than the tablet detection which barely works.
